### PR TITLE
Replace the scripts required for using socket pktio

### PR DESCRIPTION
**Pull request by bogdanPricope**
- Replace outdated scripts
- Cleanup the repo

### CODE REVIEW

File: example/README [hunk 1]
```diff
@@ -98,3 +98,19 @@ Restrictions:
 set OFP_CONFIG to OFP_CONFIG_NETWRAP_WEBSERVER
 e.g.
 #define OFP_CONFIG OFP_CONFIG_NETWRAP_WEBSERVER
+
+Environment considerations
+===============================================================================
```
Comment (matiaselo) [inline example/README:103]: I'd prefer that this section is at the top of the file so it's harder to miss by a user.

File: scripts/ofp_linux_interface_acquire.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Use this script to block access to linux interface for other
+# applications before is utilized as socket pktio in OFP
```
Comment (matiaselo) [inline scripts/ofp_linux_interface_acquire.sh:4]: Missing `it`

File: scripts/ofp_linux_interface_release.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Use this script to restore access to linux interface for other
+# applications after was blocked with ofp_linux_interface_acquire.sh.
```
Comment (matiaselo) [inline scripts/ofp_linux_interface_release.sh:4]: Typos: `a linux interface` and missing `it`

File: scripts/ofp_linux_interface_acquire.sh [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+# example applications.
+
+# Check arguments
+if [ "$#" -ne 1 ]; then
+        echo "Error: Invalid number of parameters."
+	echo "Usage:"
+	echo "  "${0}" <interface_name>"
+        exit 1
```
Comment (matiaselo) [inline scripts/ofp_linux_interface_acquire.sh:12]: Tabs and spaces mixed.

File: scripts/ofp_linux_interface_release.sh [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+# Check arguments
+if [ "$#" -ne 1 ]; then
+        echo "Error: Invalid number of parameters."
+	echo "Usage:"
+	echo "  "${0}" <interface_name>"
+        exit 1
```
Comment (matiaselo) [inline scripts/ofp_linux_interface_release.sh:11]: Tabs and spaces mixed.

**Comment by matiaselo** [comment]
Some minor comments. Otherwise looking good.

File: scripts/ofp_linux_interface_acquire.sh (hunk 4 above)
Comment (bogdanPricope) [reply scripts/ofp_linux_interface_acquire.sh:12]: OK

File: scripts/ofp_linux_interface_release.sh (hunk 5 above)
Comment (bogdanPricope) [reply scripts/ofp_linux_interface_release.sh:11]: OK

File: scripts/ofp_linux_interface_release.sh (hunk 3 above)
Comment (bogdanPricope) [reply scripts/ofp_linux_interface_release.sh:4]: OK

File: scripts/ofp_linux_interface_acquire.sh (hunk 2 above)
Comment (bogdanPricope) [reply scripts/ofp_linux_interface_acquire.sh:4]: OK

File: example/README (hunk 1 above)
Comment (bogdanPricope) [reply example/README]: ok

**Pull request merged**